lib/stack/branch/index.js

```diff
@@ -44,6 +47,29 @@ export function Branch (http, data = {}) {
      *
      */
     this.fetch = fetch(http, 'branch')
+
+    /**
+     * @description Compare allows you compare any or specific ContentType or GlobalFields.
+     * @memberof Branch
+     * @func compare
+     * @returns {Compare} Instance of Compare.
+     * @example
+     * import * as contentstack from '@contentstack/management'
+     * const client = contentstack.client()
+     *
+     * client.stack({ api_key: 'api_key'}).branch('branch_uid').compare()
```

Comment:
@nadeem-cs Change example to include branch in compare function `client.stack({ api_key: 'api_key'}).branch('branch_uid').compare('compare_branch')`